dockers/docker-iccpd/iccpd.j2

```diff
@@ -0,0 +1,11 @@
+{% for mclag_id in MC_LAG %}
+mclag_id:{{mclag_id}}
+    local_ip:{{MC_LAG[mclag_id]['local_ip']}}
+    peer_ip:{{MC_LAG[mclag_id]['peer_ip']}}
+{% if MC_LAG[mclag_id].has_key('peer_link') %}
+    peer_link:{{MC_LAG[mclag_id]['peer_link']}}
+{% endif %}
+    mclag_interface:{{MC_LAG[mclag_id]['mclag_interface']}}
+{% endfor %}
+system_mac:{{DEVICE_METADATA['localhost']['mac']}}
+
```

Comment:
Hello there - I am testing iccpd, and observet his jinja does not render in any correct data format, whitespace control might need some looking at.

```
root@leaf-1-1:/etc/iccpd# cat iccpd.conf
mclag_id:1
    local_ip:
    peer_ip:172.16.0.5
            peer_link:PortChannel3
        mclag_interface:
system_mac:cc:37:ab:17:6a:00
```

Reply:
@jpsenior, We checked the format of iccpd.j2 carefully, and can not replicate this issue.